src/createPublication.test.ts

```diff
@@ -63,28 +63,28 @@ describe("createPublications", () => {
   let store: TestStore;
   const messages = [
     createBroadcast(
-      "dsnp://0x0123456789ABCDEF",
+      "0x0123456789ABCDEF",
```

Comment:
Same question, but on the input side here. I thought we wanted users to only ever input DSNPUserURIs, not DSNPUserIds, since URIs are harder to mistake for other hex strings.

Reply:
fixed